### ISSUE TITLE
Issue with switching lights on

### ISSUE THREAD

**Issue by zavisko** [JoDehli/PyLoxone#20]
Thanks Jo for nice integration,

I have updated my component from late November to current version and I found out that switching lights on was not working for me while everything else worked fine (including switching off)

I found out that in my configuration I must have this value to be "on"
https://github.com/JoDehli/PyLoxone/blob/bb83c4b46dc626bb3be59e74c3f55570aaef24ff/custom_components/loxone/light.py#L280
so I had changed "plus" on this line to "on" and it works now fine for me

can you maybe check if that's broader issue?

thanks,
david

**Comment by JoDehli** [comment]
I will check it. My problem is that I have no lights and do not use the light controller. That is why i had no.problems. 😉 

Can you explain or send a screenshot from your light controller configuration.

**Comment by MTrobolowitsch** [comment]
Seems like turning a single Tree Spot off and on sets it to 1% brightness and doesn’t respect previous color selection, which is your fallback behaviour when no light data has been found.
Switching a light group Off/On seems to work though including scenes.

**Comment by JoDehli** [comment]
Can you test the dev branch? I changed it like you said but I could not test it without lights. 

In the dev branch I have added some more components. Maybe you can use some of them.

**Comment by zavisko** [comment]
> I will check it. My problem is that I have no lights and do not use the light controller. That is why i had no.problems. 😉
> 
> Can you explain or send a screenshot from your light controller configuration.

I quite hate loxone UI :) which is one of the reasons why i want to put HA in front of it and use loxone just as PLC.
I was using V1 lights component before. Then I found your integration and reworked my loxone to use V2, but then I found that on loxone UI all my lights were changed from simple switches to scene selectors. Basically instead of one click to turn light on/off I had to do two clicks.
Then I found out that removing all scenes/moods from default light component and keeping just the "Off" mood fixes the UI.
![lightcompo](https://user-images.githubusercontent.com/63862272/80317643-49a53800-8805-11ea-97cc-b7ff8fef5683.png)

So I think its my setup (I'm not using scenes) that requires the "on" command instead of "plus"

PS: I had a version of your integration from November which was working fine. I just updated HA last week and also updated the versions of custom_components.

PPS: when I'll find some time I'll try to develop a climate component ;). I don't see that one in your library.
